core/src/test/resources/storyWithExampleTable.story

```diff
@@ -21,4 +21,4 @@ Examples:
 Scenario: title
 Given an scenario with a table
   |this|will|be|headers|
-  |this|is  |a |row    |
\ No newline at end of file
+  |this|is  |a |row    |
```

Comment:
Don't we need a unit test that tests the variable substitution for example table rows? It might be more practical to have two different rows here